connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java

```diff
@@ -688,7 +703,7 @@ else if (value instanceof ByteBuffer)
                 }
             }
 
-            throw new DataException("Couldn't convert " + value + " to JSON.");
+            throw new DataException("Couldn't  " + value + " to JSON.");
```

Comment:
Why remove the word "convert" from this Exception? The sentence doesn't make sense without it?

Reply:
Sorry , my mistake. I have updated.